### PR TITLE
Added guide for simplest webpack commands

### PR DESCRIPTION
**Pull request by msheliga1**
This is my first open source contribution so any feedback would be appreciated.  I signed the CLA and read the writer's guide.

While I greatly appreciate the current Getting Started guide I found getting started with it to be most challenging. 

I have created a minimal setup guide that concentrates on the most basic webpack commands and features. (No lodash, npx or longer files). Featured are webpack entryfilename.js, webpack entryfilename.js -o dist/bundle.js --mode development, and an example with 3 bundled files.  Examining the bundled files and the use of development mode is discussed.  I tried to include lots of files and directory structures inside the examples.

I linked to this guide in your current getting started guide, as I felt this material was best in a separate area.  

If you can use any or all of it great.  If not I understand - I'll at least have it on my gitHub site. I do feel it would help the learning curve for folks who are new to webpack.  Any suggestions or corrections would be welcome. 

Many Thanks,

Mike

### CODE REVIEW

**Comment by montogeek** [comment]
Please sign CLA.

**Comment by msheliga1** [comment]
Would love to sign the CLA, but when I did I got the message about the
email.  I just changed my git email from name"at"domain to name@domain.  I
also changed the gitHub email settings.  I'll try to sign the CLA again,
but I'm not sure which settings need to be which in GitHub.

Many Thanks,

**Comment by msheliga1** [comment]
Fernando,

I tried to re-sign the CLA but it appears that it has already been signed.
I'm going to wait 2 days and if I don't hear anything I'm going try to
re-sign and then resubmit another pull request.  I'm pretty sure the email
issue is fixed,but can't be sure.

Many Thanks,

Mike

**Comment by montogeek** [comment]
Uhmm, push force to this PR once you set your git email to the same one that you use on GitHub.

File: src/content/guides/minimal-startup.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
+---
+title: Minimal Startup
+sort: 2
+contributors:
+  - msheliga1
+---
+
+This tutorial uses NPM, webpack and JavaScript to teach basic webpack.
+The webpack command is used to bundle and convert, or transpile, JavaScript modules.
```
Comment (EugeneHlushko) [inline src/content/guides/minimal-startup.md]: when writing commands in text, wrap them in \` to achieve: `webpack` command
Also, i can be wrong but it seems last comma isn't needed here?

File: src/content/guides/minimal-startup.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+		<br> <div id="d3">Para3 set in index.html</div>
+		<br> <script src='./dist/bundle.js'></script>
+	</body>
+</html>
+```
+
+ Edit entry.js to include require statements and update the first div.
+
+__webpack-basics/src/entry.js__
+``` JavaScript
```
Comment (EugeneHlushko) [inline src/content/guides/minimal-startup.md]: \`\`\`javascript

File: src/content/guides/minimal-startup.md [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+require('./three.js')
+console.log('Console message from entry.js')
+document.getElementById('d1').innerText='d1 set by entry.js'
+```
+
+Create two.js and thee.js in the src folder.  Edit them to include a console.log and an update of the divs.
+
+__webpack-basics/src/two.js__
+``` JavaScript
+console.log('Console message from two.js')
```
Comment (EugeneHlushko) [inline src/content/guides/minimal-startup.md]: examples are written with semicolons, and spaces for operators e.g. `var b = 1;`

File: src/content/guides/minimal-startup.md [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+
+__webpack-basics/dist/main.js__
+``` JavaScript
+!function(e){var n={};function t(r){if(n[r])return n[r].exports;var o=n[r]={i:r,l:!1,exports:{}};return e[r].call(o.exports,o,o.exports,t),o.l=!0,o.exports}t.m=e,t.c=n,t.d=function(e,n,r){t.o(e,n)||Object.defineProperty(e,n,{enumerable:!0,get:r})},t.r=function(e){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},t.t=function(e,n){if(1&n&&(e=t(e)),8&n)return e;if(4&n&&"object"==typeof e&&e&&e.__esModule)return e;var r=Object.create(null);if(t.r(r),Object.defineProperty(r,"default",{enumerable:!0,value:e}),2&n&&"string"!=typeof e)for(var o in e)t.d(r,o,function(n){return e[n]}.bind(null,o));return r},t.n=function(e){var n=e&&e.__esModule?function(){return e.default}:function(){return e};return t.d(n,"a",n),n},t.o=function(e,n){return Object.prototype.hasOwnProperty.call(e,n)},t.p="",t(t.s=0)}([function(e,n){console.log("Console.log message inside entryMain.js")}]);
```
Comment (EugeneHlushko) [inline src/content/guides/minimal-startup.md]: webpack's raw output is a subject to change with dep or and wrbpack's updates, this is not really maintainable or useful snippet

**Comment by msheliga1** [comment]
Hey folks,

thanks for picking this up and running with it.  The main idea behind the
examples was to get folks help with installing
and building the simplest example possible.  It then attempts to add some
very basic enhancements, such as a more realistic directory
structure.

I'm not sure which "getting started guide iirc" the above is referring to.
However, if it the one at https://webpack.js.org/guides/getting-started/,
I found this getting started way, way more complex and much harder to learn
from.  This is only my opinion, and others could differ.  Reasons
I feel this way include that the first example of bundling in the getting
started guide is over halfway down the page. I'm trying to show a basic
example ASAP.  The getting started guide depends upon loading additional
stuff such as lodash.  And
it shows an example of npx.  I'm not sure of all the nuances of npx, but
when I was trying to learn npm, I was looking for a "hello world" example
of npm.

Again, I appreciate your input and help.

It is up to you what you decide to do with this.  Please let me know if
there are specific changes or additional authorization you would need from
me.

Many Thanks,

Mike

**Comment by EugeneHlushko** [comment]
Hi @msheliga1 

I am not really against what you are proposing here, but would like to hear more opinions on this from other maintainers or wider audience before proceeding with adding such page.

Although, if you find `getting started guide` harder or(and) confusing, could you please try to improve it instead of creating a similar purpose page?

**Comment by snitin315** [comment]
Closing as stale and in favor of getting started guide.

**Pull request closed without merging**